frontend/src/cljs/imas_seamap/map/views.cljs

```diff
@@ -192,23 +201,27 @@
                                   (merge
                                    {:title       "No Results"
                                     :description "Try clicking elsewhere or adding another layer"
-                                    :icon        "warning-sign"}
+                                    :icon        "warning-sign"
+                                    :ref         #(when % (re-frame/dispatch [:map/pan-to-popup {:x 305 :y 210}]))}
```

Comment:
So what are these; hard-coded popup dimensions?

Reply:
Yeah. Unlike the retrieved results which are HTML elements, the feature-status popups are React elements which I can't get the width/height of. It just so happens however that their size is consistent, so there's no issue with just using a hardcoded size.

Reply:
Added comment to code for clarity.